app/main.py

```diff
@@ -1,2 +1,66 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self, initial_capacity: int = 8) -> None:
+        self.capacity = initial_capacity
+        self.size = 0
+        self.table = [None] * self.capacity
+        self.load_factor = 0.75
+
+    def _hash(self, key: Any) -> int:
```

Comment:
```suggestion
    def _hash(self, key: Hashable) -> int:
```
please fix everywhere